eventlet/patcher.py

```diff
@@ -320,6 +323,18 @@ def is_monkey_patched(module):
         getattr(module, '__name__', None) in already_patched
 
 
+def _green_existing_locks(threading):
+    if sys.version[0] != '2':
+        return
+    import eventlet.green.threading
+    lock_type = type(threading.Lock())
+    rlock_type = type(threading.RLock())
+    gc = __import__('gc')
```

Comment:
Two different ways to import in single function. Please, unify to `import` or leave a comment in code why it was necessary.